src/EA.Weee.Web/Areas/Producer/Controllers/ProducerController.cs

```diff
@@ -189,10 +190,17 @@ public async Task<ActionResult> RepresentedOrganisationDetails()
         {
             await SetViewBreadcrumb();
 
+            var source = new SmallProducerSubmissionMapperData
+            {
+                SmallProducerSubmissionData = SmallProducerSubmissionData
+            };
+
             var vm = new OrganisationDetailsTabsViewModel
             {
                 ActiveOption = OrganisationDetailsDisplayOption.RepresentedOrganisationDetails,
-                SmallProducerSubmissionData = this.SmallProducerSubmissionData
+                SmallProducerSubmissionData = this.SmallProducerSubmissionData,
+                RepresentingCompanyDetailsViewModel = SmallProducerSubmissionData.HasAuthorisedRepresentitive ? 
```

Comment:
@stevelee-civica  this wont work as the mapper is currently mapping the current submission values, we need to map what has been saved to the high level organisation data.
We need to do what we discussed this morning
![image](https://github.com/user-attachments/assets/94656791-3043-4453-aa94-23203f495f76)

Reply:
updated

Reply:
@stevelee-civica  I can't see any change to the GetSmallProducerSubmissionHandler

Reply:
pushed to please try again